Extensions/TeamCity/Src/vss-extension.json

```diff
@@ -73,7 +73,7 @@
           },
           {
             "name": "Builds",
-            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds",
+            "endpointUrl": "{{endpoint.url}}httpAuth/app/rest/buildTypes/{{definition}}/builds/?locator=branch:default:any",
```

Comment:
Strange that the API lists only builds corresponding to default branch without any locator.
@GitHubSriramB I think default version can be added incrementally to TeamCity extension, this change looks ok to list all the builds for now.